src/main/g8/src/main/resources/local-shared.conf

```diff
@@ -21,12 +21,11 @@ akka.discovery.config.services {
   "$name;format="word-only,lowercase"$" {
     endpoints = [
       {host = "127.0.0.1", port = 9101},
-      {host = "127.0.0.1", port = 9102}
+      {host = "127.0.0.1", port = 9102},
+      {host = "127.0.0.1", port = 9103}
```

Comment:
Not sure if that was an explicit choice, but node3 is missing. 

As a user, I may have `sbt -Dconfig.resource=local3.conf run` in my history and eventually start node3 as first one. Without it listed on the endpoints, it won't start the cluster.